### DISCUSSION TITLE
Stop Scan

### DISCUSSION THREAD

**Discussion by Nionor** [rommapp/romm#739] [Q&A] [answered]
Is there a way to stop the scan, in the UI I can't find a way but maybe there is a command to run in the docker?
Sometimes when a folder is added wrongly it might scan for a long time and you just see fail after fail it would be nice to be able to stop the scan. Sure I can just stop the container but that feels a bit overkill.

I guess this is a kinda beginner problem and only really matter on the initial scan or when large amount of new roms are added.

**Comment by zurdi15** [comment, marked as answer]
Hello! We already have a stop functionality built! It will be released in the next v3.1

**Reply by Nionor** [reply]
Ah great! 😃

**Comment by zblesk** [comment]
I don't think it works, though. :) I've clicked stop, but the logs showed the scan just keeps going. When I refreshed the page, it even caught on and started showing the logs. 
(I don't mind, really. I just restarted the container. :) )
